libs/barista-components/context-dialog/src/context-dialog.html

```diff
@@ -24,10 +28,6 @@
     >
       <dt-icon name="abort"></dt-icon>
     </button>
-    <ng-content select="dt-context-dialog-header"></ng-content>
-    <div class="dt-context-dialog-content" [ngClass]="overlayPanelClass">
-      <ng-content></ng-content>
-    </div>
     <ng-content select="dt-context-dialog-footer"></ng-content>
```

Comment:
Please also move the footer up in the source order. All areas where the user could add content should be before the abort button.